src/osd/OSD.cc

```diff
@@ -1024,11 +1024,13 @@ void OSDService::send_message_osd_cluster(int peer, Message *m, epoch_t from_epo
   // service map is always newer/newest
   ceph_assert(from_epoch <= next_map->get_epoch());
 
-  if (next_map->is_down(peer) ||
-      next_map->get_info(peer).up_from > from_epoch) {
-    m->put();
-    release_map(next_map);
-    return;
+  if (from_epoch < next_map->get_epoch()) {
```

Comment:
Is there any measurable impact of skipping the equality case?

Reply:
@neha-ojha . Sorry to late response.  with two storage node(every has 4NVME)+1 Client(25image)+replicated=2+fio(run=3600s, 4k Randwrite, QD=32)
w/ this patch:
write: IOPS=123k, BW=482MiB/s (506MB/s)(1696GiB/3600013msec); 0 zone resets
       lat (usec): min=316, max=203349, avg=6476.37, stdev=5404.2

w/o this patch:
 write: IOPS=120k, BW=470MiB/s (493MB/s)(1653GiB/3600027msec); 0 zone resets
lat (usec): min=322, max=162339, avg=6642.82, stdev=5695.32

Performance don't improve much.